tipocket-ctl/tpctl/build.py

```diff
@@ -32,9 +32,6 @@ def get_howto_cmds(self):
         bin_dir = self._env.dir_bin
         cmds = []
         if self._build_image:
-            for config_filepath in self._config_filepaths:
```

Comment:
We assume that user would place config file under `tipocket/config` directory by themselves, so we delete those code?

Reply:
Originally, we cp `tipocket/config/tpctl-xxx.toml` to `tipocket/build/config/tp-xxx.toml` to build a valid tipocket image.